lib/annotate/annotate_models.rb

```diff
@@ -769,6 +784,7 @@ def get_loaded_model_by_path(model_path)
     def parse_options(options = {})
       self.model_dir = split_model_dir(options[:model_dir]) if options[:model_dir]
       self.root_dir = options[:root_dir] if options[:root_dir]
+      self.skip_subdirectory_model_load = options[:skip_subdirectory_model_load].present? ? options[:skip_subdirectory_model_load] : false
```

Comment:
```suggestion
self.skip_subdirectory_model_load = options[:skip_subdirectory_model_load].present?
```
It seems to be better, I think.

Reply:
Awesome, didn't realize you could do that with `present?`